src/Symfony/Component/Messenger/Handler/MessageHandlerInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Handler;
+
+/**
+ * Handlers can implement this interface.
```

Comment:
Maybe we should say this is a marker interface somehow. Basically so it’s a bit more clear if you look in the interface, why it exists.

Reply:
I commented that we should not tell about "autoconfiguration", which isn't the incompatible with Ryan's comment :)

Reply:
👀 https://github.com/symfony/symfony/pull/26786